packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import proxyquire from 'proxyquire';
+
+import type { RootState } from '../stores/index';
+import { queries, pipelines } from '../../test/fixtures';
+import { createProxyquireMockForQueriesAndAggregationsPlugins } from '../../test/mock';
+
+const { default: AggregationsQueriesList }: any = proxyquire.load(
+  './aggregations-queries-list',
+  {
+    ...(createProxyquireMockForQueriesAndAggregationsPlugins([], []) as any),
+    react: Object.assign(React, {
+      '@global': true,
+      '@noCallThru': true,
+    }),
+  }
+);
+
+const items = [...queries, ...pipelines];
+const mockStore = configureStore<RootState>([thunk]);
```

Comment:
Absolutely not saying that you need to change anything, just FYI `createProxyquireMockForQueriesAndAggregationsPlugins` already allows you to mock items (see test in `components/index.test.tsx`) so that you don't need to mock the whole store separately (but for real it's fine like that too)